source/renderer/app/components/settings/menu/SettingsMenu.js

```diff
@@ -54,6 +68,14 @@ export default class SettingsMenu extends Component<Props> {
             active={isActiveItem(ROUTES.SETTINGS.GENERAL)}
             className="general"
           />
+          {!isSyncing && (
+            <SettingsMenuItem
+              label={intl.formatMessage(messages.stakePools)}
+              onClick={() => onItemClick(ROUTES.SETTINGS.STAKE_POOLS)}
```

Comment:
It is said that using anonymous function definition in `render` function is not so recommended.
Can we define handler method outside, and use it here?

Reply:
@yakovkaravelov Small components like this are not an issue. It's a problem in big comps, where too many re-renderings would really slow down things

Reply:
@daniloprates Yeah, that is right, that's why I said "not recommended", not "not correct".
I just wanted to use same style for any case whether it is small or big.
Anyway, not a mandatory change.... so you can leave it if you don't want to change.